pkg/config/expand.go

```diff
@@ -143,9 +144,24 @@ func (bp Blueprint) expandGroup(gp groupPath, g *Group) error {
 func (bp Blueprint) expandModule(mp ModulePath, m *Module) error {
 	bp.applyUseModules(m)
 	bp.applyGlobalVarsInModule(m)
+	// Versioned Module Logic for Embedded Modules
+	if bp.ToolkitModulesURL != "" && bp.ToolkitModulesVersion != "" {
+		if strings.HasPrefix(m.Source, "modules/") || strings.HasPrefix(m.Source, "community/") {
```

Comment:
@mr0re1 is there an opportunity here to create an `func bool isLocalModule(source)` helper? Looks like it might get used in `pkg/inspect/list.go`?